pkgs/development/python-modules/digitalocean/default.nix

```
@@ -1,23 +1,44 @@
-{ stdenv, buildPythonPackage, fetchPypi, requests, jsonpickle }:
+{ buildPythonPackage
+, fetchFromGitHub
+, pythonPackages
+, fetchPypi
+, stdenv
+}:
 
 buildPythonPackage rec {
   pname = "python-digitalocean";
-  version = "1.13.2";
+  version = "1.15.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0h4drpdsmk0b3rlvg6q6cz11k23w0swj1iddk7xdcw4m7r7c52kw";
+  src = fetchFromGitHub {
+    name = "python-digitalocean-${version}";
+    owner = "koalalorenzo";
+    repo = "python-digitalocean";
+    rev = "v${version}";
+    sha256 = "1pz15mh72i992p63grwzqn2bbp6sm37zcp4f0fy1z7rsargwsbcz";
   };
 
-  propagatedBuildInputs = [ requests jsonpickle ];
+  propagatedBuildInputs = with pythonPackages; [
+    jsonpickle
+    requests
+  ];
 
-  # Package doesn't distribute tests.
-  doCheck = false;
+  checkInputs = with pythonPackages; [
+    pytest
+    responses
+  ] ++ stdenv.lib.optionals (!isPy3k) [ mock ];
+
+  checkPhase = ''
+    cd digitalocean
+    python -m pytest
+  '';
 
   meta = with stdenv.lib; {
     description = "digitalocean.com API to manage Droplets and Images";
     homepage = https://pypi.python.org/pypi/python-digitalocean;
     license = licenses.lgpl3;
-    maintainers = with maintainers; [ teh ];
+    maintainers = with maintainers; [
+      kiwi
+      teh
+    ];
```

I think it's fine here to have it in one line. But I don't feel strongly about it
```suggestion
    maintainers = with maintainers; [ kiwi teh ];
```

---

I'm in the habit and a fan of lists being on own line I blame @worldofpeace

---

I don't feel strongly about this, you're a maintainer, I'll defer to your preference :)